### PR TITLE
Britts branch

### PR DESCRIPTION
**Pull request by BrittFreitas**
hey team, please review new components and associated code

### CODE REVIEW

**Comment by kwametsunami** [comment]
looks great! thanks for setting up sass

File: src/styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+  white-space: nowrap;
+  -webkit-clip-path: inset(100%);
+          clip-path: inset(100%);
+  clip: rect(0 0 0 0);
+  overflow: hidden;
+}
+
+h1 {
+  color: blue;
+}/*# sourceMappingURL=styles.css.map */
```
Comment (kwametsunami) [inline src/styles/styles.css:226]: approved!

**Review by kwametsunami — APPROVED** [review]

**Review by dbutch25 — APPROVED** [review]
Lovely work Britt

**Review by chriskimty — APPROVED** [review]
looks fab!!!

**Pull request merged**